plugins/database/quota/src/org/apache/cloudstack/api/response/QuotaResponseBuilderImpl.java

```diff
@@ -383,31 +383,41 @@ public QuotaTariffVO updateQuotaTariffPlan(QuotaTariffUpdateCmd cmd) {
 
     @Override
```

Comment:
There were two "addQuotaCredits" methods here, the first one with only a check. So combined the two. That removed on '@override' but still kept the '@Override' of the removed method.